org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss

```diff
@@ -473,6 +508,50 @@
       padding-left: 0 !important;
       padding-right: 0 !important;
     }
+
+    &#ContactInfo {
+      .crm-label,
+      .label {
+        display: block;
+      }
+
+      .select2-container,
+      .crm-form-text {
+        margin-right: 10px;
+      }
+    }
+
+    &#Demographics {
+      input.dateplugin {
+        border-color: $crm-darker-grayblue;
+        margin-top: 0;
+      }      
+    } 
+
+    &#CustomData {
```

Comment:
This is exceeding 4 level hierarchy plz try to  minimize this...maybe through
&#CustomData input[type=radio] {